app/views/pages/dashboard.html.erb

```diff
@@ -1,30 +1,49 @@
-<h2>Dashboard</h2>
-
-<div class="container">
-  <h3><%= @user.first_name%> <%= @user.last_name %></h3>
-  <div class="row text-center">
-    <p>Bio - <%= @user.bio %></p>
+<div class="container-fluid">
+  <!-- Top row -->
+  <div class="row d-flex justify-content-center p-3">
+    <!-- User info -->
+    <div class="grid-card col-md-10 d-flex align-items-center p-3 m-4">
+      <%= image_tag current_user.avatar, class:"img-fluid mr-4", style: "height: 150px; width: 150px; border-radius: 50%;" %>
+      <div class="container">
+      <h4>
+        <%= @user.first_name %> <%= @user.last_name %>
+      </h4>
+      <p class="text-muted">
+        <%= @user.email %> | <%= @user.location %> <br>
+        <%= @user.bio %>
+      </p>
+      </div>
+    </div>
   </div>
-</div>
-<!-- User's listed rentals -->
-<div class="container">
-  <h3>Bikes</h3>
 
-  <% @user.bikes.each do |bike| %>
-    <h4><%= bike.name %> - Total Price: $<%= bike.price %></h4>
-  <% end %>
-</div>
+  <div class="container-fluid bg-white px-5 pb-5">
+    <!-- Listing History -->
+    <div class="row d-flex justify-content-start p-3 m-4">
+      <div class="grid-card col-md-10 d-flex align-items-center p-3 m-4">
+        <i class="fas fa-bicycle mr-4"></i>
+        <h4>Listing History</h4>
+      </div>
+    </div>
+    <div class="row border m-3">
+      <% @user.bikes.each do |bike| %>
+        <div class="card-product m-1">
+          <%= cloudinary_imgs(bike.images[0].key,bike) %>
+          <div class="card-product-infos d-flex flex-column justify-content-start">
+            <h2 class="card-title"> <%= bike.name %> </h2> <hr>
+            <p><i class="fas fa-dollar-sign"></i> <%= bike.id %></p> <hr>
+            <%= link_to "More Info", bike_path(bike), class:"mb-3"%>
+          </div>
+        </div>
+      <% end %>
+    </div>
 
-<!-- User's upcoming rentals -->
-<div class="container my-5">
-<h3>Rentals</h3>
-<div class="row text-center text-lg-left">
-  <% @user.bookings.each do |booking| %>
-    <div class="col-lg-3 col-md-4 col-6 card-booking-category">
-        <%= booking.till %>
-        <%= link_to 'Add a review', new_booking_review_path(booking) %>
-      </a>
+  </div>
+  <!-- Rental History -->
+    <div class="row d-flex justify-content-center p-3">
+    <div class="grid-card col-md-12 d-flex align-items-center p-3 m-4">
+      <i class="fas fa-list-alt mr-4"></i>
+      <h4>Rental History</h4>
```

Comment:
Below could go something like below here ( but in a card )and the renters avatar ontop )  :   
Do we want users to have a public profile page ? like => https://www.spinlister.com/profile/18873

[ # user bookings card block sample ]
```
<% @user.bookings.each do |booking| %>

    <div class="col-lg-3 col-md-4 col-6 card-booking-category">
        <%= booking.bike %>
        <%= image_tag booking.bike.images[0]
, class:"img-fluid mr-4", style: "height: 150px; width: 150px; border-radius: 50%;" %>

        <%= booking.from %><%= booking.till %>
         <%= booking.total_price %> from User <%= booking.bike.user
 %>  <%= image_tag booking.bike.user.avatar, class:"img-fluid mr-4", style: "height: 150px; width: 150px; border-radius: 50%;" %>
        <%= link_to 'Add a review', new_booking_review_path(booking) %>
 <% end %>
```

Reply:
Nice Chris, I changed most of the look and data retrieval just then.